programs/x509/cert_write.c

```diff
@@ -456,6 +529,16 @@ int main( int argc, char *argv[] )
                 q = r;
             }
         }
+        else if( strcmp( p, "format" ) == 0 )
+        {
+            if      ( strcmp(q, "der" ) == 0 ) opt.format = FORMAT_DER;
```

Comment:
this does not match the coding style, but please don't change it, as I think this is much more readable (and personally, the only time I think `if` without braces is acceptable is if the whole statement fits on one line like this

Reply:
We're going to automatically rewrite the coding style in a couple of months, so it's not worth spending time on enforcing the current style.

However, I would generally argue against
```
if (condition) do_stuff();
```
on one line because it makes it hard or impossible to set a breakpoint on the call to `do_stuff` in debuggers. We're currently planning to keep it forbidden in the new coding style.